Sources/ArgumentsUtils.swift

```diff
@@ -129,20 +141,24 @@ let bundledTemplatesPath = Path(ProcessInfo.processInfo.arguments[0]).parent() +
  - returns: The Path matching the template to find
  */
 func findTemplate(prefix: String, templateShortName: String, templateFullPath: String) throws -> Path {
-  guard templateFullPath.isEmpty else {
+  switch (templateFullPath, templateShortName) {
+  case ("", ""): throw TemplateError.noTemplateProvided
```

Comment:
I've made the change but I am wondering if the `!` is not making this harder to understand than going for just `isEmpty` 🤔